File: week3-AppDev-crud/crud-python/db_connection.py
#!/usr/bin/env python3      
from cassandra.cluster import Cluster
from cassandra.auth import PlainTextAuthProvider

# This is the Zip file you downloaded
SECURE_CONNECT_BUNDLE = '/Users/cedricklunven/Downloads/secure-connect-devworkshopdb.zip'
# This is the username, recommended value was KVUser
USERNAME = "todouser";
# This is the password, recommended value was KVPassword
PASSWORD = "todopassword";
# This is the keyspace name, recommended value was killrvideo
KEYSPACE = "todoapp"; 

class Connection:
    def __init__(self):
        self.secure_connect_bundle=SECURE_CONNECT_BUNDLE
        self.path_to_creds=''
        self.cluster = Cluster(
            cloud={
                'secure_connect_bundle': self.secure_connect_bundle
            },
            auth_provider=PlainTextAuthProvider(USERNAME, PASSWORD)
        )
        self.session = self.cluster.connect(KEYSPACE)
    def close(self):
        self.cluster.shutdown()
        self.session.shutdown()

File: week3-AppDev-crud/crud-nodejs/db_connection.js
const cassandra = require('cassandra-driver');
const TimeUuid = require('cassandra-driver').types.TimeUuid;

// This is the Zip file you downloaded
const SECURE_CONNECT_BUNDLE = '/Users/cedricklunven/Downloads/secure-connect-devworkshopdb.zip'
// This is the username, recommended value was KVUser
const USERNAME = "todouser";
// This is the password, recommended value was KVPassword
const PASSWORD = "todopassword";
// This is the keyspace name, recommended value was killrvideo
const KEYSPACE = "todoapp"; 

// Init the connection and return the client
function init_connection(){
    var connection = {}
    connection.client = new cassandra.Client({ 
        cloud: { secureConnectBundle: SECURE_CONNECT_BUNDLE },
        keyspace: KEYSPACE,
        credentials: { username: USERNAME, password: PASSWORD } 
    });
    return connection
}

connection = init_connection()


module.exports = connection;